### ISSUE TITLE
MG::Text on_touch is not working

### ISSUE THREAD

**Issue by hugoerg56** [HipByte/motion-game#28]
```
    back = MG::Text.new("<", 'OpenSans-Regular', 60) 
    back.position = [X, Y]
    back.text_color = [1.0, 1.0, 1.0, 1.0]
    back.on_touch do |touch|
      puts 'touch event!'
    end    
```

is not working the touch event, any idea?

note: the MG::Button is working fine.

NOTE: also is not working with the MG::Layout too.

**Comment by eichert12** [comment]
See #35

**Comment by eichert12** [comment]
This is fixed with the merge of #37
